missilewars-plugin/src/main/java/de/butzlabben/missilewars/menus/MenuItem.java

```diff
@@ -122,7 +122,7 @@ private void updatePapiValues(Player player) {
     
     public static void setEnchantment(ItemStack itemStack) {
         ItemMeta itemMeta = itemStack.getItemMeta();
-        if (itemMeta != null) itemMeta.addEnchant(Enchantment.LUCK, 10, true);
+        if (itemMeta != null) itemMeta.addEnchant(Enchantment.LUCK_OF_THE_SEA, 10, true);
```

Comment:
As the sword is used as a combat weapon, `FORTUNE` makes more sense here. People never actually fish in MissilesWars. ;)